clusters/hosted-mgmt/hive/pools/rhtap/rhtap-ocp-4-15-0-amd64-aws-us-east-2_clusterpool.yaml

```diff
@@ -33,7 +33,7 @@ spec:
   pullSecretRef:
     name: pull-secret
   runningCount: 1
-  size: 10
+  size: 5
```

Comment:
after merging this PR there will be still some rhtap jobs using this cluster pool:
* [rhtap-cli](https://github.com/openshift/release/blob/591ef65bac1ba3d5d1ab1d77d0053ac968b64f90/ci-operator/config/redhat-appstudio/rhtap-cli/redhat-appstudio-rhtap-cli-main.yaml#L48-L114)
* [tssc](https://github.com/openshift/release/blob/591ef65bac1ba3d5d1ab1d77d0053ac968b64f90/ci-operator/config/redhat-appstudio/tssc-sample-pipelines/redhat-appstudio-tssc-sample-pipelines-main.yaml#L81-L111)

so I reduced the size to half for now

Reply:
after follow-up discussion we decided to scale down all "rhtap" cluster pools to zero and let all konflux and rhtap-related jobs use only the "konflux" cluster pool